kong/tools/dns.lua

```diff
@@ -27,7 +27,9 @@ local setup_client = function(conf)
     timeout = nil,                   -- taken from system resolv.conf; timeout
     badTtl = conf.dns_not_found_ttl, -- ttl in seconds for dns error responses (except 3 - name error)
     emptyTtl = conf.dns_error_ttl,   -- ttl in seconds for empty and "(3) name error" dns responses
+    staleTtl = conf.dns_stale_ttl,   -- ttl in seconds for records once they become stale
     order = conf.dns_order,          -- order of trying record types
+    noSynchronisation = conf.dns_no_sync
```

Comment:
We had a trailing comma in the previous line, and it's great since the diff was easy to read here. However, no trailing comma here 😅